python/ray/data/_internal/datasource/parquet_datasource.py

```diff
@@ -158,6 +160,21 @@ class ParquetDatasource(Datasource):
     cost of some potential performance and/or compatibility penalties.
     """
 
+    _FUTURE_FILE_EXTENSIONS = [
+        "parquet",
+        # Snappy compression
+        "parquet.snappy",
+        "snappy.parquet",
+        # Gzip compression
+        "parquet.gz",
+        # Brotili compression
+        "parquet.br",
+        # Lz4 compression
+        "parquet.lz4",
+        # Zstd compression
+        "parquet.zst",
```

Comment:
Can you help me understand where these are coming from? It should be `.snappy.parquet` for ex, not the other way around

Reply:
These are the canonical file extensions for the compression formats that PyArrow supports.

I agree that `parquet.snappy` is more common, but I've also seen `snappy.parquet`, so I included it.

How should I change this list?